content/pages/enterprise.html

```diff
@@ -0,0 +1,132 @@
+{# View this page at http://localhost:8080/pricing/enterprise/ #}
+
+<html>
+  <head>
+    <title>Read the Docs Enterprise Plans</title>
+    <meta name="template" content="page" />
+    <meta name="slug" content="pricing/enterprise" />
+    <meta name="status" content="hidden" />
+    <meta name="description" content="Need more than our self-serve plans? Our Enterprise pricing gives you more options." />
+  </head>
+  <body>
+
+{% import "partials/basic.html" as basic %}
+{% import "partials/homepage.html" as homepage %}
+
+{% macro about_divide(text=None) %}
+<div class="ui text container">
+  <div class="ui {%- if text %} horizontal {%- endif %} divider">
+    {% if text %}{{ text }}{% endif %}
+  </div>
+</div>
+{% endmacro %}
+
+  <section>
+    {% call basic.header_lead(
+    header="Read the Docs is the <em><strong>all-in-one</strong></em> solution you're looking for your documentation")
+    %}
+    <p>
+        We offer customized enterprise plans for organizations,
+        and can support many different customizations.
+        Works with our team to find the plan that's right for you.
+    </p>
+      <a class="ui center aligned padded segment button" href="/pricing/enterprise/#enterprise-inquiry">Get started on an Enterprise plan</a>
+      </div>
+    {% endcall %}
+  </section>
+
+
+
+
+<section id="additional-features">
+
+    {{ about_divide("Enterprise Features") }}
+  
+    <div class="ui very padded container" id="more">
+      <div class="ui vertically padded centered grid">
+  
+        {% call homepage.feature_detail_item(header="Custom builders", icon="fad fa-robot") %}
+            Get additional memory or CPU in your builds,
+            to support your larger documentation builds.
+        {% endcall %}
+  
+        {% call homepage.feature_detail_item(header="Support SLA", icon="fad fa-folder-tree") %}
```

Comment:
The icons on these blocks still look to need some tuning. I would poke around the Font Awesome search for some icons that work well, or there might be some already used in the about page here that might fit better.